app/views/webchat/SelfEmploymentIncomeSupportView.scala.html

```diff
@@ -31,8 +31,7 @@ <h1 @if(!isIvrRedirect){class="header-margin-top"}>@messages("self.employment.in
     <div class="govuk-inset-text">
         <p>@messages("global.opening.times.title")</p>
         <p>@messages("global.opening.times.8to7_30")</p>
-        <p>@messages("global.opening.time.sat.31july")</p>
-        <p>@messages("global.closed.time.sun1aug")</p>
+        <p>@messages("global.opening.times.weekend.p")</p>
```

Comment:
@tonymyers do we need to put this message back in the messages file? I can't see it but may have missed it. Thanks

Reply:
@lmugnai it's a common message so it's already there on line 22